test/es6/regexflags.js

```diff
@@ -57,28 +58,37 @@ function generatePrototypeFlagsTests() {
 
 var tests = [
     {
-        name: "Test sticky and unicode getter on RegExp.prototype",
+        name: "Test sticky and unicode and dotAll getter on RegExp.prototype",
         body: function () {
-            assert.isFalse(RegExp.prototype.unicode, "RegExp.prototype.unicode");
-            assert.isFalse(RegExp.prototype.sticky, "RegExp.prototype.sticky");
+            assert.areEqual(undefined, RegExp.prototype.unicode, "RegExp.prototype.unicode");
+            assert.areEqual(undefined, RegExp.prototype.sticky, "RegExp.prototype.sticky");
+            assert.areEqual(undefined, RegExp.prototype.dotAll, "RegExp.prototype.sticky");
 
-            function verifier(r, expectedUnicode, expectedSticky)
+            function verifier(r, expectedUnicode, expectedSticky, expectedDotAll)
             {
                 r.unicode = true; // no-op
                 r.sticky = true; // no-op
+                r.dotAll = true; // no-op
                 assert.areEqual(expectedUnicode, r.unicode, r + ": unicode");
                 assert.areEqual(expectedSticky, r.sticky, r + ": sticky");

```

Comment:
Looks like you could add a third assertion here for expectedDotAll

Reply:
Oops I thought I had I'll update that.